user-service/src/main/java/uk/gov/gchq/palisade/service/user/UserApplication.java

```diff
@@ -24,12 +24,20 @@
 import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
 
+/**
+ * Application entrypoint and main process runner
+ */
 @EnableDiscoveryClient
 @EnableCaching
 @SpringBootApplication
 public class UserApplication {
```

Comment:
Note changes are out of scope and may not necessarily make it depending on what is being done in  the user-service ticket

Reply:
was requrired to stop sonarQube issues. will remove once user service changes have been merged

Reply:
removed